chatbot.js

```diff
@@ -61,7 +57,21 @@ const replyToUser = (user, message) => {
         karma.giveKarma(api.postBotReply, karmaUser, -1);
       }
     }
-  }
+    else {
```

Comment:
Response if the input command does not fall under any category . Could be turned into an error message "No such command exists".